### PR TITLE
lesspass-cli: init at 9.0.0

### PR DESCRIPTION
**Pull request by jasoncarr0**
###### Motivation for this change

Adding LessPass CLI tool

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by risicle** [comment]
I managed to get the tests working - I've pushed a commit to my branch: https://github.com/risicle/nixpkgs/tree/ris-lesspass

I needed to pull the source from github as the tests aren't included in the pypi tarball.

**Comment by jasoncarr0** [comment]
Awesome work with the tests. Thanks. I was a bit worried about pulling in the github given the README, but you're right that it's fine on a release version.

File: pkgs/tools/security/lesspass/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+let
+  inherit (python3.pkgs) buildPythonApplication pytest mock pexpect;
+in
+buildPythonApplication rec {
+  pname = "lesspass";
+  version = "8.0.1";
+
+  src = fetchFromGitHub {
+    owner = "lesspass";
```
Comment (worldofpeace) [inline pkgs/tools/security/lesspass/default.nix]:
```suggestion
    owner = pname;
```

File: pkgs/tools/security/lesspass/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    repo = "lesspass";
```
Comment (worldofpeace) [inline pkgs/tools/security/lesspass/default.nix]:
```suggestion
    repo = pname;
```

File: pkgs/tools/security/lesspass/default.nix [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{ stdenv, python3, fetchFromGitHub }:
+
+let
+  inherit (python3.pkgs) buildPythonApplication pytest mock pexpect;
+in
+buildPythonApplication rec {
+  pname = "lesspass";
+  version = "8.0.1";
```
Comment (worldofpeace) [inline pkgs/tools/security/lesspass/default.nix]: 9.0.0. was released
* https://github.com/lesspass/lesspass/releases/tag/9.0.0

File: pkgs/tools/security/lesspass/default.nix [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+  checkInputs = [ pytest mock pexpect ];
+  checkPhase = ''
+    mv lesspass lesspass.hidden  # ensure we're testing against *installed* package
+    pytest tests
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Stateless password manager";
+    homepage = https://github.com/lesspass/lesspass;
```
Comment (worldofpeace) [inline pkgs/tools/security/lesspass/default.nix]:
```suggestion
    homepage = https://lesspass.com;
```

**Review by worldofpeace — CHANGES_REQUESTED** [review]
Please add yourself to the maintainers list in a separate commit like
```
maintainers: add jasoncarr
```

**Comment by jasoncarr0** [comment]
Sorry, I was traveling at the time this came in and missed it for a while.

**Comment by worldofpeace** [comment]
Can you rename it to `lesspass-cli`?
Other than that this looks good.

**Comment by jasoncarr0** [comment]
Okay, that should be good now

**Review by worldofpeace — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**